ITEvents/Screens/Search tab/ISearchPresenter.swift

```diff
@@ -1,9 +1,7 @@
-//
-//  ISearchPresenter.swift
-//  ITEvents
-//
-//  Created by Sofia on 28/01/2018.
-//  Copyright © 2018 Sofia. All rights reserved.
-//
-
-import Foundation
+protocol ISearchPresenter {
+    func setup()
+    
+    func setupBeforeViewAppear(then completion: (() -> Void)?)
```

Comment:
We looks like don't need `completion` here.

Take a look at implementation all the code is synchronous.
So we easily can do:
```
presenter.setupBeforeViewAppear()
collectionView.reloadData()
```
instead of:
```
presenter.setupBeforeViewAppear(then: { self.collectionView.reloadData() })
```